org.sonarlint.eclipse.core.tests/src/test/java/org/sonarlint/eclipse/core/internal/jobs/AnalyzeStandaloneProjectJobTest.java

```diff
@@ -108,7 +111,7 @@ public void analyzeWithRuleParameters() throws Exception {
 
     var underTest = new AnalyzeStandaloneProjectJob(new AnalyzeProjectRequest(slProject, List.of(fileToAnalyze), TriggerType.EDITOR_CHANGE));
     underTest.schedule();
-    assertThat(underTest.join(10_000, new NullProgressMonitor())).isTrue();
+    assertThat(underTest.join(100_000, new NullProgressMonitor())).isTrue();
```

Comment:
Do you really need to wait 100s?

Reply:
It will wait, at most, 100 seconds. The reason for that change is that running the unit test locally most often takes longer than 10 seconds, sometimes only 11 seconds, and sometimes more than a minute because the test workspace needs to synchronize before starting tests.